internal/namespaces/registry/v1/custom_tag.go

```diff
@@ -129,3 +131,29 @@ func tagListBuilder(c *core.Command) *core.Command {
 
 	return c
 }
+
+type customTagDeleteArgs struct {
+	*registry.DeleteTagRequest
+	timeout time.Duration
```

Comment:
Comparing with other fields in DeleteTagRequest that are exported, shouldn't `timeout`  be defined in a similar way (ie exported)? like [in this other customTag](https://github.com/scaleway/scaleway-cli/blob/704ebe3cec03270585045c2dff94a9c8de88d377/internal/namespaces/registry/v1/custom_tag.go#L30)